exchanges/btcmarkets/btcmarkets_wrapper.go

```diff
@@ -49,7 +47,7 @@ func (b *BTCMarkets) GetDefaultConfig() (*config.ExchangeConfig, error) {
 func (b *BTCMarkets) SetDefaults() {
 	b.Name = "BTC Markets"
 	b.Enabled = true
-	b.Verbose = true
+	b.Verbose = false
```

Comment:
revert to true

Reply:
All of our other `SetDefaults` use false